src/Shaolinq/TransactionContext.cs

```diff
@@ -146,6 +148,8 @@ private static TransactionContext GetOrCreateCurrent(DataAccessModel dataAccessM
 
 				return context;
 			}
+
+			dataAccessTransaction.CheckAborted();

```

Comment:
Without this check you end up hitting an `ObjectDisposedException` when using `DataAccessScope` if you try and continue using an outer scope after rolling back an inner scope. This makes it consistent with `TransactionScope`, which throws a `TransactionAbortedException`.